WordPress/src/main/res/values/strings.xml

```diff
@@ -2708,8 +2713,6 @@
 
     <!-- Posts -->
     <string name="post_list_author">Post author</string>
-    <string name="post_list_author_me">Me</string>
```

Comment:
When we remove/rename string we also remove all the translations. Even though the key of the string might not be ideal, I'd consider keeping `post_list_author` and `post_list_author_me` and introducing
<string name="page_list_author_me" translatable="false">@string/post_list_author_me</string>
...

Wdyt?

Reply:
I did not know that about strings. I went ahead and added back the Me & Everyone strings. Thanks for the tip and for teaching me.

Reply:
In commit 879ff76

Reply:
aha - I can't leave those in there because they are unused and tests will fail. Going to pull them out and remove from all translation string files. Removed in commit 6d428d4

Reply:
I'm not sure what has happened with my comment but I'm pretty sure I posted
```and introducing <string name="page_list_author_me" translatable="false">@string/post_list_author_me</string>"```

This would ensure that we re-use the existing translations for "me" and "everyone".

Reply:
I took them out. Lesson learned.